src/codemodder/codemods/base_codemod.py

```diff
@@ -107,11 +108,10 @@ def docs_module(self) -> Traversable:
 
     @cached_property
     def description(self) -> str:
-        if not self._metadata.has_description:
-            return ""
-
-        doc_path = self.docs_module / f"{self.origin}_python_{self.name}.md"
-        return doc_path.read_text()
+        if self._metadata.description == None:
```

Comment:
```suggestion
        if self._metadata.description is None:
```

Fixing this probably makes it unnecessary to ignore the type check below.